alexandria/core/serializers.py

```diff
@@ -37,16 +39,39 @@ class Meta:
         )
 
 
-class CategorySerializer(BaseSerializer):
+class SlugModelSerializer(BaseSerializer):
+    """
+    Ensure on creation that the model will recevie a slug.
+
+    If no slug is passed via API, one is derived from the `name` field of
+    the model, via Django's `slugify()` helper. If the model in question
+    does not have a `name` field, you can specify a `slug_source_field`
+    attribute on the serializer subclass responsible to tell it which
+    field to use instead.
+    """
+
+    def create(self, validated_data):
+        slug_field = getattr(self, "slug_source_field", "name")
```

Comment:
I suggest the "slug_source_field" to live on the Meta class.

Reply:
Good idea, fixing